gammapy/estimators/utils.py

```diff
@@ -25,12 +25,15 @@
 from .map.core import FluxMaps
 
 __all__ = [
-    "get_combined_significance_maps",
+    "combine_flux_maps",
+    "combine_significance_maps",
     "estimate_exposure_reco_energy",
     "find_peaks",
     "find_peaks_in_flux_map",
     "resample_energy_edges",
     "get_rebinned_axis",
+    "get_combined_flux_maps",
+    "get_combined_significance_maps",
```

Comment:
`get_combined_significance_maps` already appears in te dev doc, now the other methods also will.